### DISCUSSION TITLE
Installation Portainer

### DISCUSSION THREAD

**Discussion by MasterofKungfu9999** [camueller/SmartApplianceEnabler#454] [Installation und Betrieb]
Hallo, 

gibt es irgendwo eine Anleitung wie ich den SmartApplianceEnabler auf Portainer installieren kann? 

Ich blicke da nicht ganz durch und bräuchte eine Anleitung und Hilfe.

Herzlichen Dank.

**Comment by timre1337** [comment]
Schließe mich dem Gesuche an, komme zwar sonst ganz gut mit Portainer klar, beim SAE hapert es aber. Habe versucht die YAML für Compose zu übernehmen bzw. anzupassen aber ohne Erfolg. Falls das jemand schon zum Laufen bekommen hat via Portainer, wäre ich ebenfalls für Tipps sehr dankbar!
